src/components/CorporateCampaign/CampaignVerificationForm.vue

```diff
@@ -64,19 +65,51 @@ export default {
 		return {
 			iFrameSrc: null,
 			iFrameVisible: false,
-			iFrameWidth: 600,
-			iFrameHeight: 800,
+			iFrameWidth: 500,
+			iFrameHeight: 500,
 			spId: '',
 			spUserId: '',
 		};
 	},
 	mounted() {
-		// initiate form if it has not been recently submitted
-		if (!this.cookieStore.get('kvma-verified')) {
-			this.initVerificationForm();
+		// make sure we have a form id to fetch
+		if (this.formId) {
+			// check the db to see if this user has already submitted to this form id
+			this.checkFormSubmissionComplete();
 		}
 	},
 	methods: {
+		checkFormSubmissionComplete() {
+			this.apollo.query({
+				query: gql`query employeeVerificationFormQuery($formId: String!) {
+					getForm(formId: $formId) {
```

Comment:
Is the error below just an artifact of the Form Assembly Service not being quite ready by the time this PR was made? (It should be ready now!)